charts/cbcontainers-operator/README.md

```diff
@@ -0,0 +1,57 @@
+# cbcontainers-operator
+
+This is the official Helm chart for installation of the CBContainers Operator.
+
+## Installation
+
+The chart can be installed as is, without any customization or modifications.
+
+```sh
+helm repo add vmware TODO-chart-repo/TODO-chart-name
+helm repo update
+helm install cbcontainers-operator TODO-chart-repo/TODO-chart-name
+```
+
+or from source
+
+```sh
+cd charts/cbcontainers-operator
+helm install cbcontainers-operator ./cbcontainers-operator-chart
+```
+
+## Customization
+
+| Parameter                                              | Description                                                      | Default                                                                            |
+| ------------------------------------------------------ | ---------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| `spec.operator.image.repository`                       | The repository of the operator image                             | `cbartifactory/octarine-operator`                                                  |
+| `spec.operator.image.version`                          | The version of the operator image                                | `5.1.0`                                                                            |
```

Comment:
this is probably too dynamic to have it here as default - maybe use "latest version" or something? Or we have to change it on every release